rules/allied-infantry.yaml

```diff
@@ -382,3 +382,41 @@ jumpjet:
 		UpgradeTypes: airborne
 		UpgradeMaxEnabledLevel: 0
 	-Passenger:
+
+cleg:
+	Inherits: ^Infantry
+	Selectable:
+		Bounds: 20, 30, 0, -11
+	Tooltip:
+		Name: Chrono Legionnaire
+		Description: High-tech soldier.\n  Strong vs Infantry, Vehicles\n  Weak vs Aircraft
+	Buildable:
+		Queue: Infantry
+		BuildAtProductionType: Infantry
+		BuildPaletteOrder: 60
```

Comment:
I think this is too low. This must be go to last, as it has TechLevel=10 in original game.